tars/tasks/html/compile-templates.js

```diff
@@ -1 +1,139 @@
-module.exports = function () {};
+'use strict';
+
+const gulp = tars.packages.gulp;
+const replace = tars.packages.replace;
+const plumber = tars.packages.plumber;
+const rename = tars.packages.rename;
+const through2 = tars.packages.through2;
+const fs = require('fs');
+const notifier = tars.helpers.notifier;
+const browserSync = tars.packages.browserSync;
+const generateStaticPath = require(tars.root + '/tasks/html/helpers/generate-static-path');
+
+var patterns = [];
```

Comment:
Still don't see any reasons to use `var` instead of `let` here. Preventing vars on the linting stage could save you from making silly scope mistakes.